docs/modules/ROOT/pages/scripts.adoc

```diff
@@ -18,7 +56,7 @@ async def run(nre):
 
 NOTE: This script assumes there is an link:https://github.com/OpenZeppelin/nile/tree/main/src/nile/artifacts[Account artifact] in your project. You can add the contract to the contracts folder, and compile with `nile compile` to get the artifact.
```

Comment:
just a comment, an [alternative thing](https://github.com/OpenZeppelin/nile/issues/329#issuecomment-1364435529) is to `declarer_account.declare("Account", nile_account=True)` and it will declare nile's account artifact.

Reply:
Script updated.